.swiftformat

```diff
@@ -67,3 +68,4 @@
 --indent 4 # indent
 --anonymousforeach ignore # preferForLoop
 # --self init-only # redundantSelf TODO
+--redundanttype inferred
```

Comment:
hmmm interesting, I feel like in some cases we might want the type to be explicit

```
var number = 2 // inferred as Int
var number: Double = 2
```

or 

```
var id: UUID!
let copy = id // inferred as UUID? but we know this should have a value
```